proto/asn1/asn1-universal-types/asn1_universal_types.proto

```diff
@@ -0,0 +1,77 @@
+// This proto represents the Universal types
+// found in X509 Certifiactes.
+syntax = "proto2";
+
+package asn1_universal_types;
+
+// See X.690 (2015), 8.1.2.5.
+enum Encoding {
+  Primitive = 0;
+  Constructed = 1;
+}
+
+// See X.690 (2015), 8.6.
```

Comment:
Let's spend some time discussing how to expand these comments.

While this accurately describes where to find a definition of a BITSTRING type, I think it's worth discussing how it relates to the proto design. Your comment on line 15 is a good example of that, but I think we want to discuss further - e.g. the applicability/relevance of unused_bits

Reply:
Comments expanded to elaborate on representation of types and protobuf design decisions.